lib/pymedphys/docs/users/howto/gamma/from-dicom.ipynb

```diff
@@ -4,9 +4,9 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    axes_reference[0].sort()
</pre><p>Is this always needed? I think for demonstration purposes it is best to show what happens before doing this, and then to show how to do it safely.</p><p> </p><p>Even here, I reiterate a comment I made earlier that I'm not sure this is safe. It seems to me you run a strong risk of mapping the wrong coordinates to the dose grid voxels.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1770/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
No, for "head first supine" this .sort() is not needed.
But for other orientations, we have to .sort() for some axes (not all three).

I agree, it indeed can bring risk, but if we don't .sort() axes, gamma calculation can only be successfully performed for DICOM with "Head first supine"

Please advise!
Maybe we can add a new example showing how to deal with DICOM (non - "head first supine")?

Reply:
Let me check a few things and get back to you, sorry.